### DISCUSSION TITLE
不能开始运行脚本

### DISCUSSION THREAD

**Discussion by mengqi1230** [moesnow/March7thAssistant#79] [Q&A] [answered]
[出现错误 2147942402 (0x80070002) (启动“"D:\360安全浏览下载器\2\文件夹新建\March7thAssistant_v1.6.9\March7th Assistant.exe" main”时)]
![image](https://github.com/moesnow/March7thAssistant/assets/154420428/263c5f24-d650-4c0e-bc2b-a4c71ffbb31e)

**Comment by moesnow** [comment, marked as answer]
将小助手文件夹加入杀毒软件排除项，然后使用 Update.exe 更新一次即可。
病毒和威胁防护→管理设置→添加或删除排除项

**Reply by mengqi1230** [reply]
谢谢大佬！

**Comment by haibara-brownie** [comment]
360杀毒加了白名单也无效，运行updata就直接杀了，然后运行小助手也会杀，平时用都把360杀毒关了

**Reply by TheStarry001** [reply]
同顶,设置白名单一样会被杀

**Reply by moesnow** [reply]
可以尝试卸载360更换为火绒

**Reply by yhzcake** [reply]
杀软?傻卵!

**Comment by 656250265** [comment]
我如此操作了还是有如此提示，无法运行。难道是因为我系统是WIn11 吗？
